src/main/java/wolox/training/controllers/UserController.java

```diff
@@ -90,18 +90,24 @@ public Object getLoggedUser(Authentication authentication) {
      *
      * @return {@link UserBook} book's user
      */
-    @GetMapping("/name/{name}")
+    @GetMapping("/{name}")
```

Comment:
for this endpoint, the /name was ok, since the name param is always required

Reply:
Fixed at last commit !